Resources/views/backend/swag_tax/view/main/window.js

```diff
@@ -40,22 +40,6 @@ Ext.define('Shopware.apps.SwagTax.view.main.Window', {
 
     setData: function(data) {
         this.wizard.getForm().setValues(data);
-
-        var shopSelect = Ext.ComponentQuery.query('[name=shops]')[0],
-            store = shopSelect.getStore();
-
-        store.load({
```

Comment:
how does it work now, if I could not longer select a shop?

Reply:
yes.. you cant change this setting in the backend for SubShops.. For this reason we only change the main shop setting.. As in the backend

Reply:
okay :+1: